styles/AboutPage.css

```diff
@@ -0,0 +1,394 @@
+    margin-top: 35px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 30px 60px 30px 60px;
+  }
+
+  .logo_inner_container {
+    border: 2px solid var(--light_background);
+    padding: 50px 30px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: fit-content;
+  }
+
+  .logo_image {
+    width: 60%;
+  }
+
+  .event_sample {
+    padding: 34px 20px 34px 20px;
+    border-top: 3px solid var(--light_background);
+    margin-top: 19px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .sample_header {
+    text-align: center;
+    flex-direction: column;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .text5 {
+    font-family: 'Lato', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 28px;
+    margin-bottom: 20px;
+  }
+
+  .text6 {
+    font-family: 'Lato', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 21px;
+    margin-top: 20px;
+    color: var(--light-grey);
+  }
+
+  .past_event_container {
+    display: flex;
+    gap: 25px;
+    padding: 35px 200px 35px 200px;
+    flex-wrap: wrap;
+    width: 100%;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .pass_event_cards {
+    margin: 0;
+    position: relative;
+    max-width: 400px;
+  }
+
+  .event_details {
+    position: absolute;
+    top: 0;
+    height: 100%;
+    width: 100%;
+    background-color: var(--transparent_primary);
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 0 30px 0 30px;
+    color: var(--light);
+    z-index: 0;
+    visibility: hidden;
+  }
+
+  .pass_event_cards:hover .event_details {
+    visibility: visible;
+  }
+
+  .event_image {
+    height: 100%;
+    width: 100%;
+  }
+
+  .event-date {
+    font-style: normal;
+    font-weight: 800;
+    font-size: 25px;
+  }
+
+  .description {
+    font-family: 'Lato', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 21px;
+  }
+
+  .partners_section {
+    background-color: var(--dark_secondary);
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding: 50px 10px 50px 10px;
+    color: var(--light);
+    display: none !important;
+  }
+
+  .partners_section_header {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    margin-bottom: 10px;
+  }
+
+  .partners_logo {
+    width: 100%;
+    align-items: center;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    gap: 13px;
+    margin-top: 25px;
+  }
+
+  .company_logo {
+    width: 25%;
+    justify-content: space-between;
+  }
+
+  .footer_container {
+    display: flex;
+    flex-direction: row-reverse;
+    padding: 74px 146px 73px 221px !important;
```

Comment:
- [OPTIONAL] Kindly remove this padding to fix this weird-looking footer in some screen sizes. I have checked it without it by disabling this padding and it looks nice without padding. 
![as](https://user-images.githubusercontent.com/96683843/235316430-130d5ebc-1bd6-4565-a3fc-3c2ed0bee288.jpg)